yoda.spec

```diff
@@ -1,15 +1,18 @@
-### RPM external yoda 1.5.5
+### RPM external yoda 1.6.4
 
 Source: http://cern.ch/service-spi/external/MCGenerators/distribution/%{n}/%{n}-%{realversion}-src.tgz
+#Source: http://www.hepforge.org/archive/yoda/YODA-%{realversion}.tar.gz
 
 Requires: boost python cython
+#Requires: python cython
 
 %if "%{?cms_cxxflags:set}" != "set"
 %define cms_cxxflags -std=c++0x -O2
 %endif
 
 %prep
 %setup -q -n %{n}/%{realversion}
+#%setup -q -n YODA-%{realversion}
 
 ./configure --prefix=%i --with-boost=${BOOST_ROOT} CXX="$(which %cms_cxx)" CXXFLAGS="%cms_cxxflags"
```

Comment:
You have removed `boost` from run-time dependencies again, but still ask for it at configure time. In this case it will grab `boost` from system standard location if it's available. If package supports, explicitly disable boost at configuration time thus we don't pick it up from the standard system location.